src/tracker.ts

```diff
@@ -318,6 +319,26 @@ function addEditableTableRow(tracker: Tracker, entry: Entry, table: HTMLTableEle
 
     renderNameAsMarkdown(nameField.label, getFile, component);
 
+    let expandButton = new ButtonComponent(nameField.label)
+        .setClass("clickable-icon")
+        .setClass("simple-time-tracker-expand-button")
+        .setIcon(`chevron-${entry.collapsed ? 'right' : 'down'}`)
+        .setTooltip(entry.collapsed ? "Expand" : "Collapse")
+        .onClick(async () => {
+            if (entry.collapsed) {
+                delete entry.collapsed;
+            } else {
+                entry.collapsed = 1;
+            }
+            await saveTracker(tracker, this.app, getFile(), getSectionInfo());
+        });
+    if (!entry.subEntries?.length) expandButton.buttonEl.style.visibility = 'hidden';
+    let nameWrapper = nameField.cell.createDiv({cls: "simple-time-tracker-table-expandwrapper"});
```

Comment:
As far as I can tell, this clas on the name wrapper isn't used anywhere, so it doesn't seem necessary

(Also, what does this name wrapper actually do? I don't think I understand haha)

Reply:
I had to do some CSS wrangling in a previous commit where I had to style a wrapping div. You are right it seems like I am no longer using it in the most recent version.

I am not at the device where I have this all setup right now, but I can try out removing this wrapping div in a couple days

Reply:
Resolved by 19f37fe56ae89e7198016b318b5cf334e6dadcad.